humanmine.ipynb

```diff
@@ -331,206 +341,390 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "print(projects)"
+    "html = \"<table>\"\n",
+    "for gene in development_stage:\n",
+    "    images = development_stage[gene]\n",
+    "    if len(images) > 0:\n",
+    "        html += '<tr><td><h2>Gene: '+gene+'</h2></td></tr><tr>'\n",
+    "        for k, v in images:\n",
+    "            html += '<tr><td><h4>Developmental stage: '+k+'</h4></td></tr><tr>'\n",
+    "            for i in v:\n",
+    "                qs = {'id': i}\n",
+    "                url = IMAGE_DATA_URL.format(**qs)\n",
+    "                url_link = LINK_URL.format(**qs)\n",
+    "                html += '<td><a href=\"'+url_link+'\" target=\"_blank\"><img src=\"'+url+'\"/></a></td>'\n",
+    "            html += \"</tr>\"\n",
+    "        html += \"</tr>\"\n",
+    "html += \"</table>\"\n",
+    "display(HTML(html))"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "fallen-girlfriend",
+   "id": "367f36ee",
    "metadata": {},
    "source": [
-    "## Find the images\n",
-    "Find the images linked to the selected genes in one the project found.\n",
-    "Below, we look at the gene **PDX1**"
+    "**Now we filter images with a given Organism Part value**.\n",
```

Comment:
I just cannot find any place where the ``Organism Part`` value is being set. How does the value from ``TISSUE = "Pancreas" # "Cerebellum" # "little brain"`` which is used in Humanmine pass into the variable ``PART_KEY`` is not clear to me.

Reply:
Ah, sorry, ignore, I can see it in line ``if PART_KEY in name and (TISSUE or EXPRESSION in value)``